lib/ansible/modules/monitoring/grafana_datasource.py

```diff
@@ -92,6 +92,12 @@
     description:
       - The TLS CA certificate for self signed certificates.
       - Only used when C(tls_client_cert) and C(tls_client_key) are set.
+  tls_skip_verify:
```

Comment:
The default parameter name for this used by other modules is `validate_certs=no`, and this defaults to yes.
(87 modules are supporting this).

Other modules also use `client_cert` and `private_key`, but there are others. It would be nice if this could be better consolidated like `validate_certs`.

Reply:
the `tls_skip_verify` is used for the grafana datasource certificate validation, not the grafana api certificate validation (which is configurable with the `validate_certs` option)

Reply:
Understood. (I was only looking at the patch, so...)